clippy_utils/src/qualify_min_const_fn.rs

```diff
@@ -256,7 +263,19 @@ fn check_statement<'tcx>(
 
 fn check_operand<'tcx>(tcx: TyCtxt<'tcx>, operand: &Operand<'tcx>, span: Span, body: &Body<'tcx>) -> McfResult {
     match operand {
-        Operand::Move(place) | Operand::Copy(place) => check_place(tcx, *place, span, body),
+        Operand::Move(place) => {
+            if let [proj_base] = place.projection.as_ref()
```

Comment:
This need to handle any number of field projections as long as there is at least one (The `[_, ..]` or `[.., _]` patterns).

I was slightly wrong on an earlier comment. You don't need to check the base type of the projection specifically, only the type of the field being moved.

Reply:
Would that be done by getting the type of `place` or by using `Field(.., ty)` like it does now?

Reply:
The type of the place is what you want so long as there is a non-empty projection.